pkg/hwinfo/hwinfo_nonwindows.go

```diff
@@ -11,7 +11,20 @@ import (
 	"github.com/cloudradar-monitoring/dmidecode"
 )
 
+func isCommandAvailable(name string) bool {
+	cmd := exec.Command("/bin/sh", "-c", "command -v "+name)
```

Comment:
Does this try to execute the provided command?
Can this be a security risk?

Reply:
this is builtin shell command